Dockerfile

```diff
@@ -1,10 +1,10 @@
 # keep variables beyond the single build stages, see https://stackoverflow.com/a/53682110/12529534
```

Comment:
The built image is 128 MB on my machine, quite a lot for a base image.

We can shave off 19 MB (quite a percentage) if we delete the package lists in the last `RUN` line

```
# rest of the Dockerfile   ... \
    && rm -rf /var/lib/apt/lists/*
```

From my understanding, it is still possible to install debian packages but only after running `apt update`